core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletsMetadata.java

```diff
@@ -132,19 +134,33 @@ private TabletsMetadata buildExtents(AccumuloClient client) {
         var ranges = extents.stream().map(KeyExtent::toMetaRange).collect(toList());
         scanner.setRanges(ranges);
 
+        boolean extentsAreNull = extentsToFetch == null;
```

Comment:
Thinking the following may make the code a bit easier to read.  Could drop the nots in the if stmts.

```suggestion
        boolean extentsPresent = extentsToFetch != null;
```